src/main/java/org/folio/consortia/service/impl/TenantServiceImpl.java

```diff
@@ -56,8 +56,7 @@ public class TenantServiceImpl implements TenantService {
   private static final String TENANTS_IDS_NOT_MATCHED_ERROR_MSG = "Request body tenantId and path param tenantId should be identical";
 
   private static final String DUMMY_USERNAME = "dummy_user";
-  @Value("${folio.system-user.username}")
-  private String systemUserUsername;
+  private static final String SYSTEM_USERNAME = "mod-consortia-keycloak";
```

Comment:
In Eureka system user is created automatically with username that equals to module name. So it is not necessary now to populate correct system username in application.yml